tests/resources/opentelemetry/telegraf.conf

```diff
@@ -0,0 +1,54 @@
+# Centreon telegraf configuration
+# This telegraf configuration is generated by centreon centengine
+[agent]
+  ## Default data collection interval for all inputs
+  interval = "60s"
+
+[[outputs.opentelemetry]]
+   service_address = "127.0.0.1:4317"
+
+
+[[inputs.exec]]
+  name_override = "otel_check_icmp_host_1"
+  commands = ["/usr/lib/nagios/plugins/check_icmp 127.0.0.10"]
+  data_format = "nagios"
+  [inputs.exec.tags]
+    host = "host_1"
+    service = ""
```

Comment:
Are these tabulations correct?
On line 8, I have an impression of too many spaces, am I wrong?

Reply:
done